modules/visual_script/visual_script_editor.cpp

```diff
@@ -1040,7 +1041,7 @@ void VisualScriptEditor::_member_button(Object *p_item, int p_column, int p_butt
 			} else if (p_button==0) {
 
 				signal_editor->edit(name);
-				edit_signal_dialog->set_title(TTR("Editing Signal:")+" "+name);
+				edit_signal_dialog->set_title(TTR("Editing Signal: " + name));
```

Comment:
The `TTR()` macro substitutes translatable strings, so it must not contain any variables

Also, trailing white-space is easy to overlook when translating, so the separate space character should preferably stay outside the macro as well